### PR TITLE
update referral links

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

File: src/components/InviteFriendsIcon.js [hunk 1]
```diff
@@ -78,6 +88,9 @@ InviteFriendsIcon.defaultProps = {
     username: '',
     id: PropTypes.string,
     numUsersRecruited: 0,
+    cause: {
+      landingPagePath: '/cats/',
```
Comment (kmjennison) [inline src/components/InviteFriendsIcon.js:92]: Remove default. It should always exist.

File: src/components/InviteFriendsIcon.js [hunk 2]
```diff
@@ -70,6 +77,9 @@ InviteFriendsIcon.propTypes = {
     username: PropTypes.string,
     id: PropTypes.string,
     numUsersRecruited: PropTypes.number,
+    cause: PropTypes.shape({
+      landingPagePath: PropTypes.string,
```
Comment (kmjennison) [inline src/components/InviteFriendsIcon.js:81]: Make cause and cause.landingPagePath required

File: src/utils/urls.js [hunk 3]
```diff
@@ -49,15 +49,16 @@ export const externalNavigation = (url) => {
...
-export const baseURL = 'https://tab.gladly.io/cats/'
-export const getReferralUrl = (username) => {
+export const constructBaseUrl = (landingPagePath) =>
+  `https://tab.gladly.io${landingPagePath}`
+export const getReferralUrl = (username, landingPagePath) => {
   const referralUrl = username
-    ? `${baseURL}?u=${encodeURIComponent(username)}`
-    : baseURL
+    ? `${constructBaseUrl(landingPagePath)}?u=${encodeURIComponent(username)}`
+    : constructBaseUrl(landingPagePath)
```
Comment (kmjennison) [inline src/utils/urls.js:57]: Could you add some tests for this? Mainly, as defensive programming: if landingPagePath is undefined, make sure it correctly links to the main homepage.

File: src/pages/account.js [hunk 4]
```diff
@@ -292,7 +293,9 @@ const Account = ({ data: fallbackData }) => {
         ) : null}
         <Divider />
         <AccountItem
-          name="Leave Tab for Cats"
+          name={`Leave Tab for ${
```
Comment (kmjennison) [inline src/pages/account.js:296]: Flag with `// TODO: @workaround/tab-generalization`

File: src/pages/account.js [hunk 5]
```diff
@@ -315,9 +318,11 @@ const Account = ({ data: fallbackData }) => {
                 variant="caption"
                 className={classes.revertButtonText}
               >
-                Warning: This will remove your ability to support cats. It will
-                send you to classic Tab for a Cause, which has other (non-cat)
-                nonprofits you can support.
+                {`Warning: This will remove your ability to support ${
```
Comment (kmjennison) [inline src/pages/account.js:321]: Flag with `// TODO: @workaround/tab-generalization`

File: src/components/InviteFriends.js [hunk 6]
```diff
@@ -56,11 +61,14 @@ const InviteFriends = ({ user: { username } }) => {
...
     username: PropTypes.string,
+    cause: PropTypes.shape({
+      landingPagePath: PropTypes.string,
+    }),
   }),
 }
 
 InviteFriends.defaultProps = {
-  user: {},
+  user: { cause: { landingPagePath: '/cats/' } },
```
Comment (kmjennison) [inline src/components/InviteFriends.js:71]: Remove default value

File: src/components/InviteFriends.js [hunk 7]
```diff
@@ -56,11 +61,14 @@ const InviteFriends = ({ user: { username } }) => {
 InviteFriends.propTypes = {
   user: PropTypes.shape({
     username: PropTypes.string,
+    cause: PropTypes.shape({
+      landingPagePath: PropTypes.string,
+    }),
```
Comment (kmjennison) [inline src/components/InviteFriends.js:66]: Make these required

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: src/utils/urls.js (hunk 3 above)
Comment (amaliwanag2) [reply src/utils/urls.js:57]: ok.  I was building everything to default to cats but main homepage makes sense

**Comment by amaliwanag2** [comment]
tested locally, still building on dev

**Pull request merged**